src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const strings = sourceString.split(';');
+
+  strings.forEach((string) => {
+    const [property, value] = string.split(':');
```

Comment:
When splitting the string by ':', consider the possibility of having multiple colons within the value part of the declaration (e.g., in a content property). You might want to use a limit on the split method or a different approach to ensure that only the first colon is used to split the property from the value.